### ISSUE TITLE
Anti-AFK Setting

### ISSUE THREAD

**Issue by RushNY** [matthieugrieger/mumbledj#164]
#### Before submitting this issue, please acknowledge that you have done the following:
- [x ] I've at least skimmed through the [README](https://github.com/matthieugrieger/mumbledj/blob/master/README.md)
- [x ] I have checked that I am running the latest version of MumbleDJ (use `mumbledj --version` when starting the bot or use the MumbleDJ version command in Mumble)
- [x ] I have [searched through the existing issues](https://github.com/matthieugrieger/mumbledj/issues) to see if my issue has been answered already

---

#### What type of issue is this?
- [ ] Bug report (encountered problems with MumbleDJ)
- [x] Feature request (request for a new functionality)
- [ ] Question
- [ ] Other:

---

#### Log output of bot with `--debug` flag (likely only for bug reports):

---

#### Description of your issue: Our server moves AFK users to a certain channel after a set amount of time if they haven't spoken, it would be nice to have mumbledj "blip" every once in a while when nothing is playing so it stays in its set channel.

**Comment by matthieugrieger** [comment]
Good idea!

**Comment by ca21a7b567** [comment]
Sorry for the necro.. but can't you just add mumbledj to the exception list? If you are using Mumo, in /modules-enabled/idlemove.ini

> ; Comma seperated list of player names that will not be moved when idle (such as bots)
> whitelist = MumbleDJ

**Comment by matthieugrieger** [comment]
@ca21a7b567: Yeah, that's certainly possible. It may be nice to provide a mechanism in the bot (disabled by default) to do this, however.

**Comment by RushNY** [comment]
I rent a mumble server and the dashboard they provide does not have an option built in add exceptions.
